packages/nimble-components/src/utilities/unit-format/tests/default-unit-format.spec.ts

```diff
@@ -303,11 +303,6 @@ describe('DefaultUnitFormat', () => {
                 value: 2000000,
                 expectedFormattedValue: '2,000 x1000'
             },
-            {
-                name: 'uses unit-scaled value when deciding whether to format with leading-zero formatter',
```

Comment:
Could you explain why this test isn't needed anymore?

Reply:
This test ensured that we didn't pick the leading-zero formatter to use when the original (unscaled) number has a leading zero but the scaled number does not (e.g. 0.123.. scaled to 12.3..). We no longer have a separate formatter for numbers that have a leading zero, so this is no longer a meaningful test case.